sites/fast-component-explorer/app/preview.tsx

```diff
@@ -156,34 +242,20 @@ class Preview extends Foundation<{}, PreviewUnhandledProps, PreviewState> {
 
     private attachMappedComponents(): void {
         if (this.state.dataDictionary !== undefined && this.ref.current !== null) {
-            const designSystemProvider = document.createElement(
-                "fast-design-system-provider"
-            );
+            const root = document.createElement("div");
             const innerDiv = document.createElement("div");
             this.ref.current.innerHTML = "";
 
             innerDiv.setAttribute("style", "padding: 20px;");
-            designSystemProvider.setAttribute("use-defaults", "");
-            designSystemProvider.setAttribute(
-                "background-color",
-                FASTComponents.neutralLayerL1_DEPRECATED(
-                    Object.assign({}, fastDesignSystemDefaults, {
-                        baseLayerLuminance: this.state.theme,
-                    })
-                )
-            );
-            designSystemProvider.setAttribute(
-                "style",
-                "min-height: 100vh; min-width: 100vw;"
-            );
 
-            designSystemProvider.setAttribute("direction", this.state.direction);
+            // TODO: switch the theme based on this.state.theme
+            // using the design tokens
```

Comment:
@bheston here too, I need to switch the theme, not entirely sure how to do that with the updates. Any advice?

Reply:
Here you can use `root` as the element to set design token values, so something like `baseLayerLuminance.setValueFor(root, StandardLuminance.DarkMode)` and then set the background to one of the layer recipes. I think the way this is hooked up that's done through setting `background-color` in css to `${neutralLayerL1}`.